templates/simple-output.conf.j2

```diff
@@ -1,7 +1,20 @@
 output {
+{% for output in item.output %}
+
+# {{ output.name }} output
+{% if output.condition is defined or item.exclusive | bool %}
+{% if not loop.first and item.exclusive | bool %}else {% endif %}{% if (not loop.last and item.exclusive | bool) or not item.exclusive %}if {% endif %}{% if output.condition is defined %}{{ output.condition }} {% endif %}{
```

Comment:
Maybe there is here extra if?

Reply:
No, it seems to work that way. What's especially tricky about this part of the code is that there are mixed Ansible `if` and Logstash `if`.